apps/publications/src/app/pages/my-publications-page/my-publications-page.component.html

```diff
@@ -1 +1,29 @@
-<p>my-publications-page works!</p>
+<div class="user-theme">
+  <h1 class="page-subtitle d-flex">
+    <mat-icon class="title-icon">local_library</mat-icon>
+    <span class="ml-3 mt-1">{{'MY_PUBLICATIONS.TITLE' | translate}}</span>
+  </h1>
+  <div *ngIf="!initLoading">
+    <div class="mt-4">
+      <perun-web-apps-refresh-button (refresh)="refreshTable()"></perun-web-apps-refresh-button>
+      <button
+        mat-flat-button
+        color="warn"
+        class="mr-2"
+        [disabled]="selected.selected.length === 0"
+        (click)="removePublication()">
+        {{'AUTHOR_DETAIL.REMOVE' | translate}}
+      </button>
+      <perun-web-apps-publication-filter (filteredPublication)="filterPublication($event)"></perun-web-apps-publication-filter>
+    </div>
```

Comment:
Please create a separate placeholder "MY_PUBLICATIONS.REMOVE" for the remove button. Same in All publications.

Reply:
Placeholders edited and redirecting fixed.